packages/ag-charts-enterprise/src/components/dialog/dialog.css

```diff
@@ -0,0 +1,49 @@
+.ag-charts-dialog {
+    box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.1);
+    display: flex;
+    flex-direction: column;
+    font-size: 14px;
+    gap: 1em;
+    padding: 1em;
```

Comment:
Suggest updating `em` units to `px` values for things like padding. 

If not `px` then `rems` would not suggest using `em` except for exceptional circumstances.